PWGCF/MultiparticleCorrelations/Core/MuPa-Enums.h

```diff
@@ -62,4 +62,13 @@ enum eParticleHistograms {
   eParticleHistograms_N
 };
 
+enum eAsFunctionOf {
+  AFO_INTEGRATED = 0,
+  AFO_MULTIPLICITY = 1,
+  AFO_CENTRALITY = 2,
+  AFO_PT = 3,
+  AFO_ETA = 4,
```

Comment:
I guess you only need to consider the first assignment others' is automatic
You trusted it in the case of the number of codes :)

Reply:
True. But I did it this way intentionally, so that I have a quick and direct reminder always at hand! ;-)